app/models.py

```diff
@@ -78,6 +78,7 @@ class __mongometa__:
     _id = FieldProperty(schema.ObjectId)
     model = FieldProperty(schema.String)
     seq = FieldProperty(schema.String)
+    sharedId = FieldProperty(schema.String)
```

Comment:
If you want to do something with this in the future, it's probably a good idea to add an index for it (indexes = ... above)

Reply:
I think an index is not really needed as it's more an additional property which enables to relate other metadata in Uwazi or a seperate csv file to a classification sample.
Also, sharedIds might not be unique. They could refer to a translations of a text or simply occur by chance when having classification samples from different collections/models, etc.